src/index.html

```diff
@@ -11,6 +11,15 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form class="big-bar" data-qa="big">
+        <input class="big-bar__search" type="search"
+          placeholder="Try “Los Angeles”" data-qa="keypress">
```

Comment:
Add label

Reply:
steel havent label here <label for="searchBar"></label>

Reply:
I didn´t saw label in lessons yet. Should i make it like wrap up <input> in <label>? What is the reason?
<label>
<input class="big-bar__search" type="search"
          placeholder="Try “Los Angeles”" data-qa="keypress">
</label>

Reply:
you have already passed the topic with inputs and forms
https://mate.academy/learn/css-basics/forms-new?section=theory&videoId=4770&testTaskSlug=markup_form_elements

in the video for each block new topics and new ways of solving problems are explained, and here we correct how you use the accumulated knowledge, so please do not ignore the mentors' comments, but read and correct them or ask questions in the fe_chat if you do not understand something